pkg/features/kube_features.go

```diff
@@ -187,6 +187,11 @@ const (
 	// In flavor fungibility, the preference whether to preempt or borrow is inferred from flavor fungibility policy
 	// This feature gate is going to be replaced by an API before graduation or deprecation.
 	FlavorFungibilityImplicitPreferenceDefault featuregate.Feature = "FlavorFungibilityImplicitPreferenceDefault"
+
+	// owner: @j4ckstraw
```

Comment:
Thank you for adding a feature.

I don't think this needs a feature gate. It should be fine to emit an event without guarding this under a feature gate.

Reply:
Firstly, thank you for your review.

Previously, I wanted to deactivating workload with invalid workload-priorityclass, so I add a featuregate. Currently, it seems that deactivating workload is rather complex, so I only implement: add event and validation for workloadpriorityclass.